app/transformers/api/v3/transformer.rb

```diff
@@ -1,31 +1,33 @@
 class Api::V3::Transformer
   class << self
-    def from_request(attributes_of_payload)
-      rename_attributes(attributes_of_payload, key_mapping)
+    def from_request(payload_attributes)
+      rename_attributes(payload_attributes, from_request_key_mapping)
     end
 
     def to_response(model)
-      rename_attributes(model.attributes, inverted_key_mapping).as_json
+      rename_attributes(model.attributes, to_response_key_mapping).as_json
```

Comment:
Do we need `as_json` here ? I think `to_json` does that for us, at so `Oj.dump` would do the same.  We may be invoking `as_json` twice for every payload, worth testing at least.

Reply:
I tried removing this in https://github.com/simpledotorg/simple-server/pull/1889/commits/3fbcacc71ed1ff898d813e7d148ad544c5ecf5a5 which caused [these specs](https://resolvetosavelives.semaphoreci.com/workflows/2a444ff2-59b3-4f1d-ac4e-5d8dbdfb55cf) to fail. I haven't dug into this fully but `as_json` does something weird with timestamps.

```
[3] pry(main)> e = Encounter.first
=> #<Encounter:0x00007fcb8b256f80
....
 encountered_on: Thu, 23 Jan 2020,
 created_at: Thu, 23 Jan 2020 07:36:27 UTC +00:00,
 updated_at: Thu, 23 Jan 2020 07:36:27 UTC +00:00>
[4] pry(main)> e.as_json
=> {
 ...
 "encountered_on"=>Thu, 23 Jan 2020,
 "created_at"=>Thu, 23 Jan 2020 07:36:27 UTC +00:00,
 "updated_at"=>Thu, 23 Jan 2020 07:36:27 UTC +00:00}
[5] pry(main)> e.as_json.as_json
=> { "encountered_on"=>"2020-01-23",
 "created_at"=>"2020-01-23T07:36:27.163Z",
 "updated_at"=>"2020-01-23T07:36:27.163Z"}
```
I didn't want to change this just in case the app is relying on this behaviour, just to decrease the surface of change. Will dig into this some more.

Reply:
Couldn't figure out a fix here. Calling `Oj.add_to_json` doesn't have any effect here either. Leaving this be for now, want to avoid breaking timestamps for apps unknowingly.